qualtrics_link/views.py

```diff
@@ -165,33 +162,29 @@ def internal(request):
         logger.error('No records with the huid of {} could be found'.format(huid))
         return render(request, 'qualtrics_link/error.html', {'request': request})
 
-    icommons_api = IcommonsApi()
-
     # Check if the user can use qualtrics or not
     # the value of user_can_access is set to False by default
     # if any of the checks here pass we set user_can_access to True
     if person_details.valid_dept or person_details.valid_school or user_in_whitelist:
         user_can_access = True
     
     if user_can_access:
-        # If they are allowed to use Qualtrics, check to see if the user has accepted the terms of service    
-        agreement_id = settings.QUALTRICS_LINK.get('AGREEMENT_ID')
-        acceptance_resp = icommons_api.tos_get_acceptance(agreement_id, huid)
-
-        if acceptance_resp.status_code == 200:
-            acceptance_json = acceptance_resp.json()
-            if 'agreements' in acceptance_json:
-                length = len(acceptance_json['agreements'])
-                if length > 0:
-                    acceptance_text = acceptance_json['agreements'][0]['text']
-                    request.session['spoofid'] = huid
-                    return render(request, 'qualtrics_link/agreement.html', {'request': request, 'agreement': acceptance_text})
-            else:
-                logger.error('Received acceptance status of 200 but could not find any agreements for huid: {}'.format(huid))
-                return render(request, 'qualtrics_link/error.html', {'request': request})
-        else:
-            logger.error('huid: {}, api call returned response code other than 200 {}'.format(huid, acceptance_resp))
-            return render(request, 'qualtrics_link/error.html', {'request': request})
+        # If they are allowed to use Qualtrics, check to see if the user has accepted the terms of service
+        user_acceptance = None
+        try:
+            user_acceptance = Acceptance.objects.get(user_id=huid)
+        except Acceptance.DoesNotExist:
+            logger.info('User %s has not  accepted term of service ', huid)
+        except:
```

Comment:
Could we add in the logging of the exception?

Reply:
Done